config/locales/views/circuitverse/he.yml

```
@@ -0,0 +1,154 @@
+        post_assignment_heading: "פרסם משימה"
+        grade_assignment_heading: "דרג משימה"
+        use_interactive_circuit_heading: "השתמש במעגלים אינטראקטיביים בבלוג שלך, בחומרי לימוד או במצגות PowerPoint"
+        create_group_description: "אתה יכול ליצור קבוצה ולהוסיף את התלמידים שלך! אם התלמידים כבר רשומים ב-CircuitVerse, הם יתווספו אוטומטית. אם הם עדיין לא רשומים, הם יקבלו הזמנה להירשם. לאחר הרישום, הם יתווספו אוטומטית."
+        post_assignment_description: "כדי ליצור משימה, לחץ על כפתור 'הוסף משימה'. הזן פרטים ומועד אחרון. לאחר פקיעת המועד, המשימה תינעל אוטומטית."
+        grade_assignment_description: "דרג משימות בקלות עם תצוגה מקדימה מובנית. בחר משימה של תלמיד ובצע הערכה."
+        use_interactive_circuit_description: "וודא שהפרויקט ציבורי. לחץ על 'הטמע' כדי לקבל קוד HTML5, ולאחר מכן פשוט הטמע את המעגל."
+    tos:
+      title: "CircuitVerse - תנאי שימוש"
+      main_heading: "תנאי שירות"
+      user_agreement:
+      main_heading: "1. הסכמת משתמש"
```

YAMLlint (1.35.1)

[error] duplication of key "main_heading" in mapping

(key-duplicates)